packages/fhir-converter/src/templates/cda/DataType/Coding.hbs

```diff
@@ -25,7 +25,7 @@
   // -------------------------------------------------------------------------------------------------
 --}}
 {
-	"code":"{{code.code}}",
+	"code":"{{trim code.code}}",
```

Comment:
what is the issue this is fixing?

Reply:
code cant have trailing spaces

`"description": "A string which has at least one character and no leading or trailing whitespace and where there is no whitespace other than single spaces in the contents"`